storage/innobase/buf/buf0flu.cc

```diff
@@ -1660,7 +1660,7 @@ buf_flush_LRU_list_batch(
 	/* We keep track of all flushes happening as part of LRU
 	flush. When estimating the desired rate at which flush_list
 	should be flushed, we factor in this value. */
-	buf_lru_flush_page_count += n->flushed;
+	buf_lru_flush_page_count += (ulong) n->flushed;
```

Comment:
The `flush_counters_t` members are set elsewhere to `ulint` values from functions. Should I typecast those `ulint` values or change the return type from `ulint` to `ulong` of those functions?

For instance, 
```
		n->flushed = buf_do_flush_list_batch(buf_pool, min_n, lsn_limit);
```
`buf_do_flush_list_batch(...)` returns `ulint` instead of `ulong`.
After setting `flush_counters_t` members to `ulong`, should I typecast the return from `buf_do_flush_list_batch(...)` or change `buf_do_flush_list_batch(...)` to return `ulong`?

Reply:
For now, I've just applied the second solution of a C++ style cast.